site/docs/main/configuration.md

```diff
@@ -186,6 +187,31 @@ If present, the value of the `CONTOUR_NAMESPACE` environment variable is used as
 
 The `CONTOUR_NAMESPACE` environment variable is set via the [Downward API][6] in the Contour [example manifests][7].
 
+## Bootstrap Config File
+
+The boostrap configuration file is generated by an initContainer in the Envoy daemonset which configures the Envoy
+container to connect to Contour and receive configuration via xDS.
+
+### Arguments
+
+There are arguments that can be passed to `contour boostrap` that help configure how Envoy
```

Comment:
It'd be helpful to explicitly state that the flags customize the contents of the generated bootstrap config file, just to tie together the relationship between `contour bootstrap` and the config file itself.